recipes/backports.tempfile/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "backports.tempfile" %}
+{% set version = "1.0" %}
+{% set sha256 = "1c648c452e8770d759bdc5a5e2431209be70d25484e1be24876cf2168722c762" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - backports.weakref
+
+test:
+  imports:
+    - backports.tempfile
+
+about:
+  home: https://github.com/pjdelport/backports.tempfile
+  license: PSF 2
```

Comment:
Ideally we would package the license file as well. However one is not currently included in the sdist. Have raised PR ( https://github.com/pjdelport/backports.tempfile/pull/5 ) upstream to do this.

Reply:
Seems the `sdist` does have the license file after all. Must have missed it before. Have pushed a commit to add it to this PR. Hope that is ok.

Reply:
Sure, thanks for that! I didn't see the file either.